sql/src/test/scala/org/datasyslab/geosparksql/constructorTestScala.scala

```diff
@@ -107,11 +107,7 @@ class constructorTestScala extends FunSpec with BeforeAndAfterAll {
     }
 
     it("Read shapefile to DataFrame") {
-      var spatialRDD = new SpatialRDD[Geometry]
-      spatialRDD.rawSpatialRDD = ShapefileReader.readToGeometryRDD(sparkSession.sparkContext, shapefileInputLocation)
-      spatialRDD.analyze()
-      var shapfileDf = Adapter.toDf(spatialRDD, sparkSession)
-      shapfileDf.show()
+      DataFrameFactory.geometryDFFromShapeFile(sparkSession,shapefileInputLocation).show()
```

Comment:
@tociek Can you change the value of shapefilelocation to "shapefiles/dbf" instead of "shapefiles/polygon"? That shapefile has a dbf file which can be used to demonstrate the new feature.

Reply:
done